### DISCUSSION TITLE
How to ensure consistency between development and build in Tauri

### DISCUSSION THREAD

**Discussion by liukunvs** [tauri-apps/tauri#10307] [Q&A] [answered]
How to ensure consistency between development and build in Tauri, and to avoid issues where your app works in development but fails to build, especially when using next.js.

**Comment by FabianLars** [comment, marked as answer]
There's really no way to make them totally consistent, both the frontend (nextjs) and the backend (tauri) behave differently in dev vs build/prod. You unfortunetely have to test build mode regularly and for that i'd recommend `tauri build --debug` (output in target/debug/) as it will behave as a build/prod app but has more terminal output, a terminal/console attached on Windows (a prod app doesn't have that) and the devtools enabled so you can check better for errors.
